x-pack/platform/plugins/shared/fleet/server/tasks/sync_integrations/sync_integrations_on_remote.ts

```
@@ -180,3 +204,138 @@ export const syncIntegrationsOnRemote = async (
     }
   }
 };
+
+export const getFollowerIndexInfo = async (
+  esClient: ElasticsearchClient,
+  index: string,
+  logger: Logger
+) => {
+  if (!index) {
+    throw new IndexNotFoundError(`Follower index ${index} not found`);
+  }
+  try {
+    const res = await esClient.ccr.followInfo({
+      index,
+    });
+    if (!res?.follower_indices || res.follower_indices.length === 0)
+      throw new IndexNotFoundError(`Follower index ${index} not available`);
+
+    if (res.follower_indices[0]?.status === 'paused') {
+      throw new IndexNotFoundError(`Follower index ${index} paused`); // error code ?
```

we should return these errors in the API response top level error